src/adagrams.js

```diff
@@ -1,8 +1,125 @@
+const letterPool = {
+  alphabet: ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z'],
+  quantities: [9,2,2,4,12,2,3,2,9,1,1,4,2,6,8,2,1,6,4,6,4,2,2,1,2,1],
```

Comment:
if you're going to do this, it's probably better just to store it in an `object` instead, because that way you can much more quickly tell which letter is associated with what value.

If I told you the letter pool needed 4 more `M`s, how annoying would it be to change?